spec/factories/digital_object_imports.rb

```diff
@@ -25,6 +26,7 @@
     trait(:asset) do
       digital_object_data do
         {
+          'assign_uid' => '2f4e2917-26f5-4d8f-968c-a4015b10e50f',
```

Comment:
Along with my suggestion about creating a new, separate trait for the `assign_uid` case, no need to add `assign_uid` here on the asset.  Testing it on one digital object type should be enough, and I'd feel better keeping the asset digital_object_import uid-less so that we can continue to test that the default auto-uid-assignment scenario works.